lib/ansible/modules/network/vyos/vyos_lag_interfaces.py

```diff
@@ -0,0 +1,555 @@
+# set interfaces ethernet eth2 bond-group 'bond2'
+# set interfaces ethernet eth3 bond-group 'bond3'
+#
+- name: Delete LAG attributes of given interfaces (Note This won't delete the interface itself)
+  vyos_lag_interfaces:
+    config:
+      - name: bond2
+      - name: bond3
+    state: deleted
+#
+#
+# ------------------------
+# Module Execution Results
+# ------------------------
+#
+# "before": [
+#        {
+#            "hash_policy": "layer2",
+#            "members": [
+#                {
+#                    "member": "eth1"
+#                },
+#                {
+#                    "member": "eth2"
+#                }
+#            ],
+#            "mode": "active-backup",
+#            "name": "bond2",
+#            "primary": "eth2"
+#        },
+#        {
+#            "hash_policy": "layer2+3",
+#            "members": [
+#                {
+#                    "member": "eth3"
+#                }
+#            ],
+#            "mode": "active-backup",
+#            "name": "bond3",
+#            "primary": "eth3"
+#        }
+#    ],
+# "commands": [
+#        "delete interfaces bonding bond2 hash-policy",
+#        "delete interfaces ethernet eth1 bond-group bond2",
+#        "delete interfaces ethernet eth2 bond-group bond2",
+#        "delete interfaces bonding bond2 mode",
+#        "delete interfaces bonding bond2 primary",
+#        "delete interfaces bonding bond3 hash-policy",
+#        "delete interfaces ethernet eth3 bond-group bond3",
+#        "delete interfaces bonding bond3 mode",
+#        "delete interfaces bonding bond3 primary"
+#    ],
+#
+# "after": [
+#        {
+#            "name": "bond2"
+#        },
+#        {
+#            "name": "bond3"
+#        }
+#    ],
+#
+# After state
+# ------------
+# vyos@vyos:~$ show configuration  commands | grep bond
+# set interfaces bonding bond2
+# set interfaces bonding bond3
+
+
+"""
+RETURN = """
+before:
+  description: The configuration prior to the model invocation.
+  returned: always
+  type: list
+  sample: >
+    The configuration returned will always be in the same format
+     of the parameters above.
+after:
+  description: The resulting configuration model invocation.
+  returned: when changed
+  type: list
+  sample: >
+    The configuration returned will always be in the same format
+     of the parameters above.
+commands:
+  description: The set of commands pushed to the remote device.
+  returned: always
+  type: list
+  sample:
+    - 'set interfaces bonding bond2'
+    - 'set interfaces bonding bond2 hash-policy layer2'
+"""
+
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.vyos.argspec.lag_interfaces. \
+    lag_interfaces import Lag_interfacesArgs
+from ansible.module_utils.network.vyos.config.lag_interfaces.lag_interfaces import Lag_interfaces
+
+
+def main():
+    """
+    Main entry point for module execution
+
+    :returns: the result form module invocation
+    """
```

Comment:
Needs a `required_if` as discussed in the other PR.

Reply:
Done